terraform/projects/app-ecs-instances/ecs-iam-policy.tf

```diff
@@ -31,6 +31,12 @@ data "aws_iam_policy_document" "ecs_instance_document" {
     resources = ["*"]
 
     actions = [
+      "ec2:DescribeNetworkInterfaces",
+      "ec2:AttachNetworkInterface",
+      "ec2:AttachVolume",
```

Comment:
does this allow the instance to attach *any* volume?  is it worth trying to be a bit more restrictive?

Reply:
I think we could probably be more specific: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_AttachVolume.html

Reply:
Actually it's done in here: https://github.com/alphagov/prometheus-aws-configuration-beta/pull/8/files?utf8=%E2%9C%93&diff=unified&w=1#diff-b9fa37e9b366d987de5582400d4a8326R8

Reply:
that line doesn't restrict permissions though.  The instance could still, if compromised somehow, attach arbitrary volumes.  I just wondered if we could specify `resources` to only allow attaching the prometheus volume.